### PR TITLE
Bd-377 Bd-378 Deposit And Withdraw to Aave

### PR DESCRIPTION
**Pull request by AlbyGiaco**

### CODE REVIEW

File: contracts/utils/AaveAddressHolder.sol [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+contract AaveAddressHolder is IAaveAddressHolder {
+    address public override lendingPoolAddress;
+
+    constructor(address _lendingPoolAddress) public {
+        lendingPoolAddress = _lendingPoolAddress;
+    }
+
+    ///@notice Set the address of the lending pool from aave
+    ///@param newAddress The address of the lending pool from aave
+    function setLendingPoolAddress(address newAddress) external override {
```
Comment (RiccardoGalbusera) [inline contracts/utils/AaveAddressHolder.sol:16]: make it onlyGovernance? Or maybe delete it at all if we don't need it

File: contracts/modules/AaveModule.sol [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        LendingPool = ILendingPool(_lendingPool);
+    }
+
+    function depositToAave(address token0) public {
+        IERC20(token0).approve(address(LendingPool), 1000000);
+
+        LendingPool.deposit(token0, 1000, address(this), 0);
+    }
+
+    function withdrawToAave(address token0) public {
```
Comment (luduvigo) [inline contracts/modules/AaveModule.sol:21]: **withdrawFromAave**

**Review by RiccardoGalbusera — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -38,5 +38,8 @@
     "ts-node": "^10.1.0",
     "typechain": "^5.1.2",
     "typescript": "^4.5.2"
+  },
+  "dependencies": {
```
Comment (luduvigo) [inline package.json:42]: this should be a devDependency

File: test/shared/fixtures.ts [hunk 4]
```diff
@@ -95,3 +97,27 @@ export async function getSelectors(contract: any) {
 
   return selectors;
 }
+
+//find the slot in memory of the balance of the tokenAddress and return it
+export async function findbalanceSlot(MockToken: any, user: any) {
+  const encode = (types: any, values: any) => ethers.utils.defaultAbiCoder.encode(types, values);
+  const account = user.address;
+  const probeA = encode(['uint'], [10]);
```
Comment (luduvigo) [inline test/shared/fixtures.ts:105]: Naming is terrible here

Comment (luduvigo) [reply test/shared/fixtures.ts:105]: probeA, probeB, probedSlot, prev ...

**Review by luduvigo — CHANGES_REQUESTED** [review]

**Review by luduvigo — APPROVED** [review]

**Pull request merged**